### ISSUE TITLE
[5.0] Dark mode - Drag and drop text colour contrast-

### ISSUE THREAD

**Issue by C-Lodder** [joomla/joomla-cms#41970]
### Steps to reproduce the issue

Go to : `administrator/index.php?option=com_installer&view=install`

### Actual result

The text highlighted in red fails the WCAG minimum required colour contrast requirements:

![image](https://github.com/joomla/joomla-cms/assets/2019801/e26c908d-ccbf-4e4d-a0fc-34a4eaf69e32)

**Comment by brianteeman** [comment]
see #42053

**Comment by richard67** [comment]
Closing as having a pull request. Please test #42053 . Thanks in advance.
